packages/ast-spec/src/token/PunctuatorToken/PunctuatorTokenToText.ts

```diff
@@ -0,0 +1,65 @@
+import * as ts from 'typescript';
+
+const SyntaxKind = ts.SyntaxKind;
```

Comment:
can't this just be 
```suggestion
import type {SyntaxKind} from 'typescript';
```

Reply:
Copied it over from `node-utils`, but could be only the type import indeed! 👍